banking-bench/src/main.rs

```diff
@@ -219,6 +274,18 @@ fn main() {
                 .takes_value(false)
                 .help("Disable forwarding messages to TPU using QUIC"),
         )
+        .arg(
+            Arg::new("simulate_mint")
+                .long("simulate-mint")
+                .takes_value(false)
+                .help("Simulate mint transactions to have hjigher priority"),
+        )
+        .arg(
+            Arg::new("mint_txs_percentage")
+                .long("mint-txs-percentage")
+                .takes_value(true)
```

Comment:
add in `.requires("simulate_mint")` so this isn't used if `--simulate-mint` isn't present.